src/llfs/volume_trimmer.test.cpp

```diff
@@ -371,7 +371,8 @@ class VolumeTrimmerTest : public ::testing::Test
     std::vector<char> buffer(data_size, 'a');
     std::string_view data_str{buffer.data(), buffer.size()};
 
-    auto&& payload = llfs::PackableRef{llfs::pack_as_raw(data_str)};
+    llfs::PackAsRawData pack_as_raw{data_str};
+    auto&& payload = llfs::PackableRef{pack_as_raw};
```

Comment:
Looks like there is similar usage in Volume::append() where we are passing the return value of pack_as_raw to PackableRef. Also, the fix uses a new variable 'pack_as_raw' which could get confused with the existing function with the same name.